jina/main/parser.py

```diff
@@ -170,6 +170,8 @@ def set_flow_parser(parser=None):
                     help='type of the output')
     gp.add_argument('--output-path', type=argparse.FileType('w', encoding='utf8'),
                     help='output path of the flow')
+    gp.add_argument('--skip-dryrun', action='store_true', default=False,
```

Comment:
Would better name it `- -skip-dry-run`

Reply:
https://helm.sh/docs/chart_template_guide/debugging/ they guys from helm also called it `- -dry-run`